### PR TITLE
Updated Privacy Whitepaper

### PR DESCRIPTION
**Pull request by jasonstephen15**
AB#45050955

### CODE REVIEW

File: microsoft-edge/privacy-whitepaper/index.md [hunk 1]
```diff
@@ -306,7 +306,7 @@ Image Descriptions can be turned on for the current web page without enabling fo
 <!-- ====================================================================== -->
 ## Image Enhancement
 
-To provide a better browsing experience, Microsoft Edge offers Image Enhancement by improving color, lighting, contrast, and sharpness of images. When Image Enhancement is turned on, Microsoft Edge encrypts and transmits images to Microsoft servers to perform image enhancement. No user identifiers are included in the requests to the servers. The images are cached for 30 days to improve performance. 
+To provide a better browsing experience, Microsoft Edge offers Image Enhancement by improving color, lighting, contrast, and sharpness of images. When Image Enhancement is turned on, Microsoft Edge encrypts and transmits image URLs to Microsoft servers to perform image enhancement. No user identifiers are included in the requests to the servers. The images are cached for 7 days to improve performance.
```
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:309]:
```suggestion
To provide a better browsing experience, Microsoft Edge offers Image Enhancement by improving color, lighting, contrast, and sharpness of images. When Image Enhancement is turned on, Microsoft Edge encrypts and transmits image URLs to Microsoft servers to perform image enhancement. No user identifiers are included in the requests to the servers. The image URLs are cached for 7 days to improve performance. 
```
what's cached: images, or image urls?

**Review by mikehoffms — CHANGES_REQUESTED** [review]
Writer/Editor pass done

**Review by mikehoffms — APPROVED** [review]

File: microsoft-edge/privacy-whitepaper/index.md [hunk 2]
```diff
@@ -306,7 +306,7 @@ Image Descriptions can be turned on for the current web page without enabling fo
 <!-- ====================================================================== -->
 ## Image Enhancement
 
-To provide a better browsing experience, Microsoft Edge offers Image Enhancement by improving color, lighting, contrast, and sharpness of images. When Image Enhancement is turned on, Microsoft Edge encrypts and transmits images to Microsoft servers to perform image enhancement. No user identifiers are included in the requests to the servers. The images are cached for 30 days to improve performance. 
+To provide a better browsing experience, Microsoft Edge offers Image Enhancement by improving color, lighting, contrast, and sharpness of images. When Image Enhancement is turned on, Microsoft Edge encrypts and transmits image URLs to Microsoft servers to perform image enhancement. No user identifiers are included in the requests to the servers. The image URL and enhanced image are cached for 7 days to improve performance.
```
Comment (dandclark) [inline microsoft-edge/privacy-whitepaper/index.md:309]:
```suggestion
To provide a better browsing experience, Microsoft Edge offers Image Enhancement by improving color, lighting, contrast, and sharpness of images. When Image Enhancement is turned on, Microsoft Edge encrypts and transmits image URLs to Microsoft servers to perform image enhancement. No user identifiers are included in the requests to the servers. The image URLs and enhanced images are cached for 7 days to improve performance. 
```
I think plural works better here.

**Review by dandclark — APPROVED** [review]
LGTM with grammar nit.

**Review by apryzgoda — CHANGES_REQUESTED** [review]
I suggest removing the duration of the cache, i.e. 7 days. We try to not use exact numbers or make promises in the public white paper, unless absolutely necessary.

**Comment by apryzgoda** [comment]
> I suggest removing the duration of the cache, i.e. 7 days. We try to not use exact numbers or make promises in the public white paper, unless absolutely necessary.

After talking with Jason, he confirmed that the "7 day" declaration is necessary to keep customers' trust. We will update the public whitepaper if that declaration should be changed.

**Review by apryzgoda — APPROVED** [review]
Edge Privacy Approved

**Review by captainbrosset — APPROVED** [review]

**Pull request merged**